adyen.php

```diff
@@ -294,13 +342,21 @@ public function displayForm()
                     'hint' => $this->l('Must correspond to the notification password in the Adyen Backoffice under Settings => Notifications')
                 ),
                 array(
-                    'type' => 'text',
+                    'type' => 'password',
                     'label' => $this->l('HMAC key for notifications'),
                     'name' => 'ADYEN_NOTI_HMAC',
                     'size' => 20,
-                    'required' => true,
+                    'required' => false,
                     'hint' => $this->l('Must correspond to the notification HMAC Key in the Adyen Backoffice under Settings => Notifications => Additional Settings => HMAC Key (HEX Encoded)')
                 ),
+                array(
+                    'type' => 'text',
```

Comment:
should it be 'password' like the other encrypted fields?

Reply:
was thinking about it but we expose it in the descripiton under it so not necessary